engine-standalone-storage/src/sync/mod.rs

```diff
@@ -183,11 +225,28 @@ pub fn consume_message(storage: &mut crate::Storage, message: Message) -> Result
             };
             storage.set_transaction_included(tx_hash, &tx_included, &diff)?;
 
-            Ok(())
+            let outcome = TransactionIncludedOutcome {
+                hash: tx_hash,
+                info: tx_included,
+                diff,
+            };
+            Ok(ConsumeMessageOutcome::TransactionIncluded(outcome))
         }
     }
 }
 
+pub enum ConsumeMessageOutcome {
+    BlockAdded,
+    FailedTransactionIgnored,
+    TransactionIncluded(TransactionIncludedOutcome),
+}
+
+pub struct TransactionIncludedOutcome {
+    pub hash: aurora_engine_types::H256,
+    pub info: crate::TransactionIncluded,
+    pub diff: crate::Diff,
```

Comment:
Nit, I shudder when I see `crate::` in the code, better to just include them at the top.